set.mm

```diff
@@ -651997,6 +651997,281 @@ of all sets ( ~ inex1g ).
     ( ccom wss wceq id coeq12d sseq12d cleq2lem ) AADZAEBBDZBEABCABFZKLABMABABM
     GZNHNIJ $.
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Additions for square root; absolute value
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+
+This is based on the observation that the real and imaginary parts of a complex
+number can be calculated from the number's absolute and real part and the sign
+of its imaginary part.
+
+<HTML>
+Formalization of the formula in ~ sqrtcval was motivated by a short
+<A HREF="https://www.youtube.com/watch?v=QAhWTQ4Ravw">Michael Penn video</A>.
+</HTML>
+$)
```

Comment:
Please respect the conventions for "Sectioning and section headers" (see https://us.metamath.org/mpeuni/conventions-comments.html) for the body of the section header comment: [blank line; header comment indented with two spaces; blank line;].

Reply:
This has been addressed throughout my sandbox. [47b6591](https://github.com/metamath/set.mm/pull/3993/commits/47b65911b0e91b3ab8d8e96b1dc0cbecf37547d8)